GameConfigurationRegister/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/gameconfigurationregister/sendingconfig/GameConfigurationQueueListener.java

```diff
@@ -0,0 +1,28 @@
+package pl.jakubowskiprzemyslaw.tajgertim.gameconfigurationregister.sendingconfig;
+
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.stereotype.Service;
+import pl.jakubowskiprzemyslaw.tajgertim.models.configuration.GameConfiguration;
+import pl.jakubowskiprzemyslaw.tajgertim.services.LoggerService;
+
+@Service
+class GameConfigurationQueueListener {
+
+    private final LoggerService loggerService;
+    private final ApplicationEventPublisher publisher;
+
+    @Autowired
```

Comment:
is it necessary?

https://docs.spring.io/spring/docs/4.3.x/spring-framework-reference/htmlsingle/#beans-autowired-annotation

Reply:
done